templates/lxd/index.html

```diff
@@ -0,0 +1,350 @@
+        alt="",
+        width="71",
+        height="60",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+      </div>
+      <a href="https://linuxcontainers.org/lxd/docs/master/">Read the documentation </a>
+    </div>
+    <div class="col-3">
+      <div class="u-sv1 u-hide--medium u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/3aef5bf9-forum-icon.svg",
+        alt="",
+        width="60",
+        height="60",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+      </div>
+      <a href="https://discuss.linuxcontainers.org/">Discuss with other users</a>
+    </div>
+    <div class="col-3">
+      <div class="u-sv1 u-hide--medium u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/3769b4a7-webinar-icon.svg",
+        alt="",
+        width="73",
+        height="60",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+      </div>
+      <a href="https://www.youtube.com/c/LXDvideos/videos">Watch tutorials on YouTube</a>
+    </div>
+  </div>
+</section>
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-6">
+      <h2 class="u-sv2">Enhance your experience with best-in-class open-source tools</h2>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-4">
+      <h3 class="p-heading--4">Deploy workloads across different platforms with Juju</h3>
+      <p>Juju makes it easy to deploy the exact same software on a variety of different platforms, including public and private cloud, bare metal, Kubernetes clusters. It is also a useful tool for local deployments, ideal for development.</p>
+      <p>For operating your instances using Juju please read <a href="https://juju.is/docs/olm/lxd">this tutorial</a>.</p>
+    </div>
+    <div class="col-4">
+      <h3 class="p-heading--4">Easily manage bare metal infrastructure with MAAS and LXD</h3>
+      <p>MAAS can automatically deploy and configure LXD as part of the deployment of a physical machine. Or it can be connected to an existing LXD deployment to dynamically create virtual machines on it.</p>
+      <p>To set up your own MAAS and LXD environment please read <a href="https://maas.io/tutorials/build-a-maas-and-lxd-environment-in-30-minutes-with-multipass-on-ubuntu#1-overview">this tutorial</a>.</p>
+    </div>
+    <div class="col-4">
+      <h3 class="p-heading--4">Deploy workloads across different platforms with Juju</h3>
+      <p>Juju makes it easy to deploy the exact same software on a variety of different platforms, including public and private cloud, bare metal, Kubernetes clusters. It is also a useful tool for local deployments, ideal for development.</p>
+      <p>For operating your instances using Juju please read <a href="https://juju.is/docs/olm/lxd">this tutorial</a>.</p>
+    </div>
+  </div>
+</section>
+<section class="p-strip--suru-blog-header">
+  <div class="row">
```

Comment:
```suggestion
  <div class="u-fixed-width">
```
`row` should only be used with nested `col-`s